test/dotnet-trx-merge.IntegrationTests/DotNetTrxMergeTests.cs

```diff
@@ -40,7 +40,7 @@ public void DotnetTrxMerge_WithSelectingFiles_Success()
         output.Should().Contain("Found 2 files to merge");
         output.Should().Contain("Found 1 tests in file");
         output.Should().Contain("Found 2 tests in file");
-        output.Should().Contain("New result of test 3dacbae9-707e-1881-d63c-3573123dffc6 was found in file ");
+        output.Should().Contain("New result of test 3dacbae9-707e-1881-d63c-3573123dffc6 - 631e6252-66d2-49b8-9fb3-6b9fc02425db was found in file ");
```

Comment:
should maybe consider testing no other 'new result of test' things in here, as we expect only one change